docs/automatic-channel-matching-algorithm.md

```
@@ -73,27 +73,33 @@ Channels to propagate: [Voltage]
 
 ### Phase 4: Validation
 
-Before suggesting channels, the system validates they exist in the destination file:
+Before suggesting channels, the system validates they exist in the destination file **by channel NAME, not by ID**:
 
 ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)